src/index.js

```diff
@@ -85,11 +85,11 @@ function sendPayment (params) {
  * @param {Object} params.destinationMemo - Memo to be included in the transfer credit of the recipient
  * @param {Object} params.sourceMemo - Memo to be included in the transfer debit coming from the sender's account
  * @param {Object} params.additionalInfo
- * @param {Condition} params.receiptCondition - Object, execution condition.
+ * @param {Condition} params.receiptCondition - Object, notary case execution condition.
  *                                              If not provided, one will be generated.
- * @param {Condition} params.executionCondition - Object, execution condition.
+ * @param {Condition} params.executionCondition - Object, transfer execution condition.
  *                                              If not provided, one will be generated.
```

Comment:
I'm having a hard time coming up with a better suggestion but this distinction between `receiptCondition` and `executionCondition` is a bit confusing. It is especially so when looking at it from the universal mode perspective where there is only one, and actually putting in either here will work.